packages/block-library/src/image/edit.native.js

```diff
@@ -270,6 +270,12 @@ class ImageEdit extends React.Component {
 			);
 		}
 
+		// We still want to render the caption so that the soft keyboard is not forced to close on Android
```

Comment:
Nit: even though the comment makes sense in the context of this PR (the change from not rendering at all to rendering with `display: none`) it's not so clear when read on its own in the final file version.

What do you think about this instead:
> We'll render the caption so that the soft keyboard is not forced to close on Android but still hide it by setting its `display` style to `none`.